pkg/collector/corechecks/systemd/systemd.go

```diff
@@ -194,6 +200,8 @@ func (c *SystemdCheck) Run() error {
 		return err
 	}
 	sender.Commit()
+
+	c.submitVersion(conn)
```

Comment:
This is the first thing we should do I think. That way if we have an issue (missing metrics, early error ...) we will still send the version to the backend.

Reply:
Sure 👍